src/content/drafts/vue-rules-better-life.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Vue Uncategorized rules to have a better life
+teaser: "Some useful Uncategorized Vue rules that will make your daily life a little easier."
+authors:
+  - BobToninho
+categories:
+  - Case Studies
+tags:
+  - Guest Post
+  - Rules
+---
+
+## Introduction
```

Comment:
We don’t need a heading here. The first few paragraphs always considered the introduction. 
```suggestion
```